### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

File: src/Components/Navbar/Navbar.css [hunk 1]
```diff
@@ -85,11 +86,24 @@
 	display: none;
 }
 
-.header .navbar {
+/* .header .navbar {
```
Comment (MuskanChhatrasal) [inline src/Components/Navbar/Navbar.css:89]: Remove commented code.

File: src/Pages/Authentication/Signup.js [hunk 2]
```diff
@@ -154,7 +154,6 @@ const Signup = () => {
 				</div> */}
```
Comment (MuskanChhatrasal) [inline src/Pages/Authentication/Signup.js:154]: Here also

File: src/Pages/Authentication/authentication.css [hunk 3]
```diff
@@ -100,9 +98,8 @@ form .text h3 a:hover {
 
 .icon {
 	position: absolute;
-	    z-index: 10;
-	    margin-top: 16px;
+	z-index: 10;
+	margin-top: 16px;
```
Comment (MuskanChhatrasal) [inline src/Pages/Authentication/authentication.css:102]: Try to use relative units like rem

File: src/Pages/Home/home.css [hunk 4]
```diff
@@ -119,3 +119,7 @@
 	color: var(--text-color-2);
 	line-height: 1.7;
 }
+
+.h3_plain,.p_plain{
+	background-color:var(--bg-color-3) !important ;
```
Comment (MuskanChhatrasal) [inline src/Pages/Home/home.css:124]: Avoid the use of important keyword

**Review by MuskanChhatrasal — APPROVED** [review]
Amazing UI with readable code, just suggested few things try to implement those else everything looks good to merge

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything seems nice to me You can go ahead and merge it

**Pull request merged**